public/main.css

```diff
@@ -0,0 +1,84 @@
+.bg-1 {
+  background-color: rgb(74, 115, 129);
+  color: #ffffff;
+}
+.bg-2 {
+background-color: rgb(74, 115, 129);
+color: #ffffff;
+}
+.bg-3 {
+  background-color: white;
+  color: #555555;
+}
+
+.address p {
+  line-height: .5;
+}
+
+.bg-3 img {
+  width: 3%;
+  height: 3%;
+  margin-top: 20px
+}
+
+.bg-0 {
+  background-color: rgb(108, 152, 167);
+  color: rgb(32, 28, 28);
+}
+
+.row {
```

Comment:
Careful about doing this sort of thing. The CSS framework assumes that internal classes like `.row` and `.column` behave a certain way. Overriding their default behavior can have unpredictable consequences.